lib/tasks/hosts.rake

```diff
@@ -34,14 +34,15 @@ namespace :hosts do
     missingdns = []
     offline = []
 
-    Host.out_of_sync(1.hour.ago).all(:order => 'environment_id asc').collect do |host|
+    Host::Managed.out_of_sync.order('environment_id ASC').collect do |host|
```

Comment:
the idea behind bumping the interval was to avoid those machines that might be running long puppet runs, maybe we should default to puppet_interval*2 instead?